### ISSUE TITLE
it doesn't work if you just have https

### ISSUE THREAD

**Issue by colttt** [andersju/webbkoll#36]
Hello,

it looks like it always try http, also when I explicit set https://example.com, so I don't use http, just https - so it don't work for me

**Comment by andersju** [comment]
By design it always tries http:// first, to see if it redirects to https:// automatically. Are you saying you're trying to check a site that _only_ uses HTTPS (i.e., one that doesn't listen to port 80 at all)? We have some crude handling for that case, but it doesn't always work. If you have an example of a site that doesn't work, let me know (here or anders@unix.se).

**Comment by colttt** [comment]
did you receive my email?

**Comment by andersju** [comment]
Yes. Haven't had time to look into it until now (doing this in my spare time). As mentioned, the way we handle HTTPS-only sites (which are extremely few) is crude and doesn't always work. When I have some time I will make the "always check http:// first" thing optional.

In the meantime, if you really want to use Webbkoll on that particular site, I suggest adding a redirect from http:// to https://. From a security point of view it doesn't matter (and it's better for usability, as browsers still default to http:// if you omit the protocol).
